### PR TITLE
Verify

### PR DESCRIPTION
**Pull request by jcrugzz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sudo: false
+language: node_js
+node_js:
+  - "8"
+  - "6"
+before_install:
+  - "npm install -g npm@latest-5"
```
Comment (3rd-Eden) [inline .travis.yml:7]: Isn't `npm@5` already latest on 6 & 8?

File: index.js [hunk 2]
```diff
@@ -66,6 +72,10 @@ Warehouse.prototype.publish = function publish(params, fn) {
   );
 };
 
+Warehouse.prototype.verify = function v(opts, fn) {
```
Comment (3rd-Eden) [inline index.js]: I would be delighted if this would receive some JSDoc.

**Review by 3rd-Eden — APPROVED** [review]
Just 2 comments, but not enough to block. LGTM

File: .travis.yml (hunk 1 above)
Comment (indexzero) [reply .travis.yml:7]: Yes to `node@8`, but not `node@6` iirc.

**Review by indexzero — APPROVED** [review]
👍  to the JSDoc comment from @3rd-Eden

**Pull request merged**